### PR TITLE
LPS-144033 Remove the product dropdown

### PR DESCRIPTION
**Pull request by ilzamcmed**

### CODE REVIEW

File: modules/apps/site-initializer/site-initializer-raylife/src/main/zippableResources/fragments/raylife/fragments/business-home-cover/index.js [hunk 1]
```diff
@@ -139,46 +135,28 @@ getQuoteForm.onsubmit = function (event) {
...
-		if (!formProps.product) {
-			productErrorFeedback.innerHTML =
-				'<span class="form-feedback-indicator"></span> Please select a product';
-			productContainer.classList.add('has-error');
-		}
 	}
 	else {
 		localStorage.setItem(
 			'raylife-product',
 			JSON.stringify({
```
Comment (kevenleone) [inline modules/apps/site-initializer/site-initializer-raylife/src/main/zippableResources/fragments/raylife/fragments/business-home-cover/index.js:158]: `JSON.stringify(formProps)`

Could be passed directly

**Review by kevenleone — CHANGES_REQUESTED** [review]

File: modules/apps/site-initializer/site-initializer-raylife/src/main/zippableResources/fragments/raylife/fragments/business-home-cover/index.js (hunk 1 above)
Comment (ilzamcmed) [reply modules/apps/site-initializer/site-initializer-raylife/src/main/zippableResources/fragments/raylife/fragments/business-home-cover/index.js]: Done

**Comment by kevenleone** [comment]
ci:forward

**Comment by kevenleone** [comment]
Just starting reviewing :)

**Comment by kevenleone** [comment]
resent: #572

**Pull request closed without merging**